dbux-projects/src/dataLib/PathwaysDataProvider.js

```diff
@@ -55,11 +56,12 @@ class ApplicationCollection extends Collection {
   serialize(application) {
     const { entryPointPath, createdAt } = application;
     const relativeEntryPointPath = path.relative(this.dp.manager.config.projectsRoot, entryPointPath).replace(/\\/g, '/');
+    const serializedDpData = JSON.stringify(application.dataProvider.serializeJson());
     return {
       relativeEntryPointPath,
       createdAt,
       uuid: application.uuid,
-      serializedDpData: application.dataProvider.serialize()
+      serializedDpData
```

Comment:
Quick check: Was this change also tested?

Reply:
Checked